cmake/makefile-variables.in

```diff
@@ -114,9 +114,6 @@ PYTHONFRAMEWORKINSTALLDIR=
 MACOSX_DEPLOYMENT_TARGET=@CMAKE_OSX_DEPLOYMENT_TARGET@
 #export MACOSX_DEPLOYMENT_TARGET
 
-# Options to enable prebinding (for fast startup prior to Mac OS X 10.3)
-OTHER_LIBTOOL_OPT=
-
```

Comment:
Commit integrated in https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/commit/ac6c28ee12feff743a0bca1b2477c447ca4a1885